bin/metrics-mongodb.rb

```diff
@@ -155,41 +155,165 @@ def run
+    server_metrics['asserts.errors'] = server_status['asserts']['msg']
+    server_metrics['asserts.regular'] = server_status['asserts']['regular']
+    server_metrics['asserts.user'] = server_status['asserts']['user']
+    server_metrics['asserts.rollovers'] = server_status['asserts']['rollovers']
 
+    # Background flushing
+    server_metrics['backgroundFlushing.flushes'] = server_status['backgroundFlushing']['flushes']
+    server_metrics['backgroundFlushing.total_ms'] = server_status['backgroundFlushing']['total_ms']
+    server_metrics['backgroundFlushing.average_ms'] = server_status['backgroundFlushing']['average_ms']
+    server_metrics['backgroundFlushing.last_ms'] = server_status['backgroundFlushing']['last_ms']
+
+    # Connections
     server_metrics['connections.current'] = server_status['connections']['current']
     server_metrics['connections.available'] = server_status['connections']['available']
+    server_metrics['connections.totalCreated'] = server_status['connections']['totalCreated']
+
+    # Cursors
+    server_metrics['clientCursors.size'] = server_status['cursors']['clientCursors_size']
+    server_metrics['cursors.open'] = server_status['cursors']['totalOpen']
+    server_metrics['cursors.pinned'] = server_status['cursors']['pinned']
+    server_metrics['cursors.totalNoTimeout'] = server_status['cursors']['totalNoTimeout']
+    server_metrics['cursors.timedOut'] = server_status['cursors']['timedOut']
+
+    # Journaling (durability)
+    server_metrics['journal.commits'] = server_status['dur']['commits']
+    server_metrics['journaled_MB'] = server_status['dur']['journaledMB']
+    server_metrics['journal.timeMs.writeToDataFiles'] = server_status['dur']['timeMs']['writeToDataFiles']
+    server_metrics['journal.writeToDataFilesMB'] = server_status['dur']['writeToDataFilesMB']
+    server_metrics['journal.compression'] = server_status['dur']['compression']
+    server_metrics['journal.commitsInWriteLock'] = server_status['dur']['commitsInWriteLock']
+    server_metrics['journal.timeMs.dt'] = server_status['dur']['timeMs']['dt']
+    server_metrics['journal.timeMs.prepLogBuffer'] = server_status['dur']['timeMs']['prepLogBuffer']
+    server_metrics['journal.timeMs.writeToJournal'] = server_status['dur']['timeMs']['writeToJournal']
+    server_metrics['journal.timeMs.remapPrivateView'] = server_status['dur']['timeMs']['remapPrivateView']
+
+    # Extra info
+    server_metrics['mem.heap_usage_bytes'] = server_status['extra_info']['heap_usage_bytes']
+    server_metrics['mem.pageFaults'] = server_status['extra_info']['page_faults']
+
+    # Global Lock
+    server_metrics['lock.totalTime'] = server_status['globalLock']['totalTime']
+    server_metrics['lock.queue_total'] = server_status['globalLock']['currentQueue']['total']
+    server_metrics['lock.queue_readers'] = server_status['globalLock']['currentQueue']['readers']
+    server_metrics['lock.queue_writers'] = server_status['globalLock']['currentQueue']['writers']
```

Comment:
these lock queue metrics were previously `lock.queue.thing`. can you change it back to that so we maintain backwards compatibility.